resources/views/Checkout/OrderConfirmation.twig

```diff
@@ -45,12 +45,30 @@
 
                     </div>
                 </div>
-
                 {{ LayoutContainer.show("Ceres::OrderConfirmation.BeforeOrderDetails", data.order) }}
 
-
                 {% include "Ceres::Checkout.OrderDetails" with { orderData: data } %}
-
+                {% if data.order.statusId >= 7 %}
+                    <div class="row">
+                        <div class="col-md-6">
+                            <div class="row">
+                                <div class="widget-inner col-12 col-md-4 widget-prop-md-3-2"></div>
+                                <div class="widget-inner col-12 col-md-4 widget-prop-md-3-2"></div>
+                                <div class="widget-inner col-12 col-md-4 widget-prop-md-3-2">
+                                    <div>
+                                        <div class="widget widget-link widget-primary order-return">
+                                            <a rel="" href="/returns/{{ data.order.id }}/"
```

Comment:
What happens when the return page is configured via shopbuilder and category ? 

I think then we have mismatch between urls. Check if you can use ShopUrls here.

Reply:
You are right. I managed to find a solution for the mismatch between urls and used {{ urls.returns(data.order.id) }}